repos/system_upgrade/el7toel8/actors/postgresqlcheck/libraries/library.py

```diff
@@ -0,0 +1,81 @@
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import CalledProcessError, api, run
+
+def get_postgresql_packages():
+    """ Get list of installed PostgreSQL packages """
+
+    for rpm_pkgs in api.consume(InstalledRedHatSignedRPM):
+        return [pkg for pkg in rpm_pkgs.items if pkg.name.startswith('postgresql')]
+
+def get_postgresql_extensions():
+    """ Get list of PostgreSQL extensions """
+    result = []
+    try:
+        output = run([
+            '/usr/bin/su',
+            '-l', 'postgres'
+            '-c', 'psql -tAc \'\dx\''
+        ], split=True)['stdout']
+    except CalledProcessError:
+        api.current_logger().debug('Unable to list PostgreSQL extensions: psql failed to run')
```

Comment:
Are you sure that should be a debug message?

Reply:
Changed to 'warning' and pushed as fixup.